src/components/cylc/tree/Tree.vue

```diff
@@ -160,7 +160,7 @@ import TreeItem from '@/components/cylc/tree/TreeItem'
 import Vue from 'vue'
 import TaskState from '@/model/TaskState.model'
 import Task from '@/components/cylc/Task'
-import clonedeep from 'lodash.clonedeep'
+import cloneDeep from 'lodash/cloneDeep'
```

Comment:
I think this is fixed in the table view PR, and probably also in another PR. Whoever gets in first wins, and the other PRs can then be updated :smile: